README.md

```diff
@@ -28,3 +31,45 @@ npm run dev
 To fill out a form and view a supported Art Blocks artwork, navigate to `http://localhost:5173/on-chain-generator-viewer/`.
 
 To view an Art Blocks artwork directly, navigate to `http://localhost:5173/on-chain-generator-viewer/<contract-address>/<token-id>`, replacing <contract-address> and <token-id> with the address of the NFT contract and the ID of the token you want to view. The application will fetch the html for the NFT and display it within an iframe on a new tab.
+
+## Hosting
+
+The application is hosted on centralized servers as well as on the decentralized IPFS network.
+
+### Pin the On-Chain Generator
+
+Help support decentralized access to the Art Blocks on-chain generator by pinning it to IPFS.
+
+IPFS detail:
+
+- CID: `QmYr3B1TREbgbS29hWmEAcnU96msFoGJ18mYemLtPkj7T6`
+- IPNS: `onchain-generator.artblocks.io`
+
+#### What is Pinning?
+
+Pinning ensures the generator interface remains accessible through the IPFS network. The more people who pin the content, the more resilient and decentralized the network becomes.
+
+#### Pin Using Pinata
+
+1. Create an account at pinata.cloud
+2. Navigate to the "Pin By CID" page
+3. Enter CID: `QmYr3B1TREbgbS29hWmEAcnU96msFoGJ18mYemLtPkj7T6`
```

Comment:
I don't know where we left off - are we using an ipns to enable folks to auto-update to latest `e.g. ipfs-cluster-ctl pin add --ipns /ipns/<peer-id>`?
or are we sticking with publishing the latest CID here for now?